deploy/vagrant/vbox/buildbox.sh

```diff
@@ -12,9 +14,16 @@ if [ -f tachyon-dev.box ]; then
   rm -f tachyon-dev.box
 fi
 
-vagrant up
-vagrant package --output tachyon-dev.box default
-vagrant destroy -f
+BASE_BOX='chef/centos-7.0'
+AVAIL_BOX=$(vagrant box list | grep $BASE_BOX | grep '(virtualbox, 1.0.0)')
+
```

Comment:
@chen-tian We probably don't want the tachyon-dev.box to include tachyon and ufs.  Each time user can specify a different source for Tachyon, or a different ufs.  Tachyon source can be a locally shared directory between the host and the vm, when it is configured to fetch tachyon from local.

For the most part, we use ansible to provision everything. That's what we do on aws.   The virtual box image is just an optimization to save time. (similarly we could create an amazon ami to save launch time, but we have not done that) 

I agree the old code is broken because it forces the user to download java, maven and install them twice. I still think something like `if not installed, install maven` would have fixed the double install problem, without losing the ability to reuse virtualbox images, but I can see why this approach may be cleaner, since we are always  building from a known state that is centos7 image.

I do think launch speed is important because the vagrant script is the quickest and the only way to test with an underfile system on a dev machine. If we can ensure relaunch speed with another script, then I am happy!